README.md

```diff
@@ -38,138 +39,157 @@ Start by cloning this repository:
+a message, or add it as a bookmark in a channel of the workspace you selected.
+Interacting with this link will run the associated workflow.
 
-After selecting a Workspace, the output provided will include the link trigger
-Shortcut URL. Copy and paste this URL into a channel as a message, or add it as
-a bookmark in a channel of the Workspace you selected.
+**Note: triggers won't run the workflow unless the app is either running locally
+or deployed!**
 
-**Note: this link won't run the workflow until the app is either running locally
-or deployed!** Read on to learn how to run your app locally and eventually
-deploy it to Slack hosting.
-
-## Running Your Project Locally
+### Manual Trigger Creation
 
-While building your app, you can see your changes propagated to your workspace
-in real-time with `slack run`. In both the CLI and in Slack, you'll know an app
-is the development version if the name has the string `(local)` appended.
+To manually create a trigger, use the following command:
 
 ```zsh
-# Run app locally
-$ slack run
-
-Connected, awaiting events
+$ slack trigger create --trigger-def triggers/<YOUR_TRIGGER_FILE>.ts
 ```
 
-Once running, [previously created Shortcut URLs](#create-a-link-trigger)
-associated with the `(local)` version of your app can be used to start
-workflows.
+## Datastores
 
-To stop running locally, press `<CTRL> + C` to end the process.
+For storing data related to your app, datastores offer secure storage on Slack
+infrastructure. The use of a datastore requires the
+`datastore:write`/`datastore:read` scopes to be present in your manifest.
 
 ## Testing
 
-Test filenames should be suffixed with `_test`. Run all tests with `deno test`:
+Test filenames should be suffixed with `_test`.
+
+Run all tests with `deno test`:
 
 ```zsh
 $ deno test
 ```
 
 ## Deploying Your App
 
-Once you're done with development, you can deploy the production version of your
-app to Slack hosting using `slack deploy`:
+Once development is complete, deploy the app to Slack infrastructure using
+`slack deploy`:
 
 ```zsh
 $ slack deploy
 ```
 
-After deploying, [create new link triggers](#create-a-link-trigger) for the
-production version of your app (not appended with `(local)`). Once the triggers
-are invoked, the associated workflows should run just as they did when
-developing locally.
+When deploying for the first time, you'll be prompted to
+[create a new link trigger](#creating-triggers) for the deployed version of your
+app. When that trigger is invoked, the workflow should run just as it did when
+developing locally (but without requiring your server to be running).
 
-### Viewing Activity Logs
+## Viewing Activity Logs
 
-Activity logs for the production instance of your application can be viewed with
-the `slack activity` command:
+Activity logs of your application can be viewed live and as they occur with the
+following command:
 
 ```zsh
-$ slack activity
+$ slack activity --tail
 ```
 
 ## Project Structure
```

Comment:
Since this is the blank template, should we adjust this so that it reflects the project's structure? Or should we keep this in as the standard recommended structure and make a note of that here?